core/templates/dev/head/services/SearchService.ts

```diff
@@ -167,10 +167,6 @@ oppia.factory('SearchService', [
         // 3. language codes (default to 'en')
         var querySegments = urlQuery.split('&');
 
-        if (querySegments.length > 3) {
```

Comment:
We have already removed this code [here](https://github.com/oppia/oppia/pull/6768/files#diff-afb3bbc3134da6ddbab1022d677aad55L170), so how it's still there?